package.json

```diff
@@ -39,22 +39,13 @@
     "protobufjs@npm:^7.2.5": "patch:protobufjs@npm%3A6.11.4#~/.yarn/patches/protobufjs-npm-6.11.4-af11968b80.patch"
   },
   "scripts": {
-    "start:docker": "cd contract && docker compose up -d",
-    "stop:docker": "cd contract && docker compose down",
-    "docker:logs": "cd contract; docker compose logs --tail 200 -f",
-    "docker:bash": "cd contract; docker compose exec agd bash",
-    "docker:make": "cd contract; docker compose exec agd make -C /workspace/contract",
-    "make:help": "make -C contract list",
-    "start:contract": "cd contract && yarn start",
-    "integration-test": "cd contract && yarn ava ./test/orca-multichain.test.js",
     "start:ui": "cd ui && yarn dev",
     "format": "yarn prettier --write .github contract ui",
     "lint:format": "yarn prettier --check .github contract ui",
     "lint": "yarn lint:format && yarn workspaces foreach --all run lint",
-    "lint:fix": "yarn format && yarn workspaces foreach --all run lint:fix",
     "test": "yarn workspaces foreach --all run test",
-    "build": "yarn workspaces foreach --all run build",
-    "postinstall": "npx patch-package"
```

Comment:
not sure if there are any repercussions for removing this. Will let @Jovonni confirm this.

Reply:
@mujahidkay brought them back.